### DISCUSSION TITLE
How to delete my Github Page

### DISCUSSION THREAD

**Discussion by quzhengpeng** [community/community#50099] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

https://github.com/quzhengpeng/quzhengpeng.github.io

I cant not visit github page repository. 
When I visit the repository, it show 404  GitHub Page.
Now I want to manage it or delete it, but I cant not do this.
![image](https://user-images.githubusercontent.com/13043954/225274753-465bf9ce-06ac-48da-83d7-c0cc5d5645b6.png)

**Comment by Anmol-Baranwal** [comment, marked as answer]
It is working fine, kindly check again.
It takes some time to deploy in the GitHub pages. 
Do reply if the issue is resolved

**Reply by quzhengpeng** [reply]
I last commited the repository about 1 year ago, it's ok, but today I visit the repository it don't work. 
You visit the repository is ok? what happened to me? Wrong DNS or others?

**Reply by Anmol-Baranwal** [reply]
![image](https://user-images.githubusercontent.com/74038190/225350672-2be7d22c-0bba-4feb-ad08-6276a5a5397f.png)

It doesn't show 404 error page, so I believe It is working fine. 
I have attached a screenshot for a reference.

**Reply by quzhengpeng** [reply]
I can visit the page https://quzhengpeng.github.io/
But I don't want to visit this page. 
I want to visit the github repository page https://github.com/quzhengpeng/quzhengpeng.github.io
such as https://github.com/quzhengpeng/panda.zsh-theme
But I can't.
![image](https://user-images.githubusercontent.com/13043954/225373326-66d7c595-9d52-48ac-a788-1d0876843b45.png)

**Reply by Anmol-Baranwal** [reply]
![image](https://user-images.githubusercontent.com/74038190/225378168-01b71660-9be9-456b-a832-a08fad4f9038.png)

As you can see, it is public repo so I can actually visit this without any issue.

Are you not able to visit the repo?

**Reply by quzhengpeng** [reply]
I'm not able to visit the repo.
But I don't know why.

**Reply by quzhengpeng** [reply]
It's working until I stop my VPN!
I think it parse wrong URL.

**Comment by datagrains** [comment]
Go here:
https://github.com/quzhengpeng/quzhengpeng.github.io/settings

Scroll down to Danger Zone and choose "Delete this repository"

![image](https://user-images.githubusercontent.com/30798870/225397472-b575c3d2-df62-4279-aa8a-807544ba4075.png)


And do the same for your other repo:
https://github.com/quzhengpeng//panda.zsh-theme/settings

Does that work?

**Reply by quzhengpeng** [reply]
I can't visit the page https://github.com/quzhengpeng/quzhengpeng.github.io/settings
When I visit this page, it show 404 page.
I think the URL parse a Github Page URL, not Github repo URL.
How can I change that?

**Reply by quzhengpeng** [reply]
Like this
![image](https://user-images.githubusercontent.com/13043954/225509279-371dbf34-1896-4b08-8973-15a3493039f2.png)
